azerothcore/azerothcore-wotlk
# fix(Scripts/Misc): Replace some AddItem/CreateItem with spells

* cherry-pick commit (https://github.com/TrinityCore/TrinityCore/commit/ed38206031f25fec73282044293238c9573a89d9)

Co-Authored-By: offl <11556157+offl@users.noreply.github.com>



## Changes Proposed:
-  Replace some AddItem/CreateItem with spells

## Tests Performed:

- none

## How to Test the Changes:


1. Check if you get rocket pack from Gunship.
2. Check if you get items from Bag of Heart Candies
3. Check if you get items from Mingo's Fortune Giblets
4. See if you get Arcane Disruptor from Chromie in Culling of Stratholme


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
